### PR TITLE
Migrate device-provisioning-service to repository

### PR DESCRIPTION
**Pull request by Danielius1922**

### CODE REVIEW

File: charts/plgd-hub/values.yaml [hunk 1]
```diff
@@ -3105,3 +3105,244 @@ m2moauthserver:
...
+  podSecurityContext: {}
+  # -- Labels for dps pod
+  podLabels: {}
+  # -- Annotations for dps pod
+  podAnnotations: {}
+  service:
+    # -- Service type
+    type: LoadBalancer
+    # -- Use nodePort, if specified, for one of the protocols. If both protocols are enabled, nodePort needs to be configured directly in the service to mutually different ports.
+    nodePort: 5684
```
Comment (jkralik) [inline charts/plgd-hub/values.yaml]: set 15684 because 5684 is used by coapgateway

File: device-provisioning-service/config.yaml [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+        cacheExpiration: 30s
+    readTimeout: 8s
+    readHeaderTimeout: 4s
+    writeTimeout: 16s
+    idleTimeout: 30s
+clients:
+  storage:
+    cacheExpiration: 10m
+    mongoDB:
+      uri: mongodb://localhost:27017
```
Comment (jkralik) [inline device-provisioning-service/config.yaml]: set uri to same value as is in other services.

**Review by jkralik — COMMENTED** [review]
We need use also enablers here:
https://github.com/plgd-dev/hub/blob/adam/feature/dps/charts/plgd-hub/templates/http-gateway/config.yaml#L55-L56

**Review by jkralik — APPROVED** [review]

**Comment by Danielius1922** [comment]
@coderabbitai create PR description for non plgd developers

**Comment by Danielius1922** [comment]
@coderabbitai create a PR description focusing on describing device provisioning service features for non-PLGD developers.

**Pull request merged**